dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/scheduling/JobSchedulerController.java

```diff
@@ -106,7 +105,6 @@ public List<SchedulerEntry> getQueueableJobs(@RequestParam(required = false) Str
             : config -> !name.equals(config.getQueueName());
     return jobConfigurationService.getAllJobConfigurations().stream()
         .filter(JobConfiguration::isConfigurable)
-        .filter(not(JobConfiguration::isLeaderOnlyJob))
```

Comment:
FYI: This logic was backwards but since user created job configurations before this change were created with `leaderOnlyJob=false` (which was wrong) this had to filter like this. But then I figured this entire filter does not make sense to have given we already filter  `isConfigurable` which means only user created jobs will be possible to select.